lib/SGN/Controller/AJAX/BrAPI.pm

```diff
@@ -1643,11 +1669,11 @@ sub lists : Chained('brapi') PathPart('lists') Args(0) : ActionClass('REST') { }
 sub lists_GET {
 	my $self = shift;
 	my $c = shift;
-	my ($auth,$user_id) = _authenticate_user($c,1);
+	my ($auth,$user_id) = _authenticate_user($c);
```

Comment:
Add a config to require auth or not

Reply:
this specific call will require to force authentication since lists belong to users. Omitting this may cause to return only public lists.